apps/export/permissions.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import permissions
+
+
+class ExportViewSetPermission(permissions.BasePermission):
+    def has_permission(self, request, view):
+        if (view.action in ('list', 'update', 'destroy') or (request.user and request.user.is_authenticated)):
+            return True
+        return False
```

Comment:
After updating `get_for` we would not need any additional permission other than `is_authenticated`. So we should remove this. @annshress let me know if I am missing something. @k9845 by this, use previous permission_classes in the viewset.

Also `(view.action in ('list', 'update', 'destroy') or (request.user and request.user.is_authenticated))` can you explain why is this here?